From 2ae83cf0f2c780d4e8b39bcdf29a65f680a6914c Mon Sep 17 00:00:00 2001
From: Kihwal Lee <kihwal@apache.org>
Date: Tue, 4 Oct 2016 11:16:43 -0500
Subject: [PATCH 2079/2748] HDFS-10878.
 TestDFSClientRetries#testIdempotentAllocateBlockAndClose
 throws ConcurrentModificationException.
 Contributed by Rushabh S Shah. Modified
 CHANGES.txt (cherry picked from commit
 382307cbdd94107350fe6fad1acf87d63c9be9d6)

(cherry picked from commit 3eff0873f609ff574db27956d6740a87c7b9ea02)

 Conflicts:
	hadoop-hdfs-project/hadoop-hdfs/CHANGES.txt

Change-Id: I70ae398d4bd385bcbf72da951c4a8193a02582c4
---
 .../apache/hadoop/hdfs/TestDFSClientRetries.java   |    4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/TestDFSClientRetries.java b/hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/TestDFSClientRetries.java
index c432043..e90ee0a 100644
--- a/hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/TestDFSClientRetries.java
+++ b/hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/TestDFSClientRetries.java
@@ -104,7 +104,6 @@
 import org.mockito.invocation.InvocationOnMock;
 import org.mockito.stubbing.Answer;
 
-import com.google.common.base.Joiner;
 
 /**
  * These tests make sure that DFSClient retries fetching data from DFS
@@ -468,8 +467,7 @@ public Boolean answer(InvocationOnMock invocation) throws Throwable {
           // complete() may return false a few times before it returns
           // true. We want to wait until it returns true, and then
           // make it retry one more time after that.
-          LOG.info("Called complete(: " +
-              Joiner.on(",").join(invocation.getArguments()) + ")");
+          LOG.info("Called complete:");
           if (!(Boolean)invocation.callRealMethod()) {
             LOG.info("Complete call returned false, not faking a retry RPC");
             return false;
-- 
1.7.9.5

